ingest/src/commands/pages.ts

```diff
@@ -10,23 +10,23 @@ type PagesCommandArgs = {
   source: string | string[];
 };
 
-const commandModule: CommandModule<PagesCommandArgs> = {
+const commandModule: CommandModule<
+  Record<string, unknown>,
+  PagesCommandArgs
+> = {
   command: "pages",
   builder(args) {
-    return args.string("source");
+    return args.string("source").demandOption("source");
```

Comment:
This is actually a bit more complicated than it seems, because we'll need to make sure we don't delete the pages that aren't in the result set just because they haven't been updated. Will follow up later as it is not essential to MVP imo. https://jira.mongodb.org/browse/DOCSP-31206